CHANGELOG.md

```diff
@@ -13,6 +13,13 @@
 - Add `signal` option to allow use of an `AbortSignal` for complexity control.
   Enables the algorithm to abort after a timeout, manual abort, or other
   condition.
+- Add `maxWorkFactor` to calculate a deep iteration limit based on the number
+  of non-unique blank nodes. This defaults to `1` for roughly O(n) behavior and
+  will handle common graphs. It must be adjusted to higher values if there is a
+  need to process graphs with complex blank nodes or other "poison" graphs. It
+  is recommeded to use this paramter instead of `maxDeepIterations` directly.
```

Comment:
```suggestion
  is recommended to use this parameter instead of `maxDeepIterations` directly.
```

Reply:
Should state that `maxDeepIterations` is provided, then `maxWorkFactor` will be ignored.